README.md

```diff
@@ -1,19 +1,17 @@
 # navitia_model [![Build Status](https://travis-ci.org/CanalTP/navitia_model.svg?branch=master)](https://travis-ci.org/CanalTP/navitia_model)
 
-
 navitia_model is a rust crate to manage transit data. Its model corresponds to the one used in [navitia](https://github.com/CanalTP/navitia). This repository also provides (incomplete) [GTFS](http://gtfs.org/) to [NTFS](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fr.md) and (soon) NTFS to GTFS conversion.
 
-## Usage
+## Compile
 
-Compile
 ```bash
 cargo build --release
 ```
 
-### Converting from GTFS to NTFS
+## Converting from GTFS to NTFS
 
 NTFS needs a `Dataset` and a `Contributor`.
-Default ones are provided by the command but you can pass a json file that contain some informations for creating a `Dataset` and a `Contributor`.
+Default ones are provided by the command but you can pass a json file that contain some informations for creating a `Dataset` and a `Contributor` as explained in the [documentation](src/documentation/gtfs2ntfs.md).
```

Comment:
containS

Reply:
and `information` is never plural (si review of PR 69)